### ISSUE TITLE
`secrets` context in `steps.with.args` wrongly detected as not allowed

### ISSUE THREAD

**Issue by Natim** [rhysd/actionlint#497]
Action lint wrongly detects `secrets` context as unusable in the `step.with.args` property:

```
on:
  schedule:
    - cron: 0 */12 * * *
  workflow_dispatch:

jobs:
  translations-pull-request:
    name: Update translations pull request
    runs-on: ubuntu-24.04
    permissions:
      contents: write
      id-token: write
      pull-requests: write
    steps:
      - name: Lokalise upload django.po fr
        uses: docker://lokalise/lokalise-cli-2:v3.0.2
        with:
          args: >-
            lokalise2 file upload
            --config lokalize.yml
            --file ./app/locale/fr/LC_MESSAGES/django.po
            --lang-iso="fr"
            --cleanup-mode="true"
            --distinguish-by-file
            --token ${{ secrets.LOKALISE_API_TOKEN }}
```

![image](https://github.com/user-attachments/assets/bf6c9379-b36a-4609-90f4-81e4be199546)

**Comment by rhysd** [comment]
This is a duplicate of #495. It is already fixed at the main branch.

**Comment by Natim** [comment]
Thank you @rhysd I will wait for the next release then.

**Comment by rhysd** [comment]
I'm going to make the next release this weekend.

**Comment by rhysd** [comment]
[v1.7.6](https://github.com/rhysd/actionlint/releases/tag/v1.7.6) for fixing this issue was released.

**Comment by Natim** [comment]
Thanks @rhysd
